VictorBevz/HW6/task2.py

```diff
@@ -0,0 +1,6 @@
+login = input("Enter your login:")
```

Comment:
good job
look at more optimal version of code
login = input("Enter your login:")

while login != "First":
    login = input("Try agai and enter your login:")
else:
    print("Login is correct!")